### DISCUSSION TITLE
Fuel Instrument Junction Boxes

### DISCUSSION THREAD

**Discussion by CaptainBobSim** [CaptainBobSim/The-Cessna-172-Project-V3#174] [General]
I have a question @Kovinator - What do these do?
![image](https://user-images.githubusercontent.com/76851401/209986484-bc36957c-fd29-4573-ad5b-1599dbeeb231.png)

**Comment by Kovinator** [comment]
They are optional “junction” boxes to combine common wires. You can take a pin header and solder them together to create a “bridge” that can then be hot glued in place inside each box. The one to the farthest right is common negative (5 wires total - 2 for led cathode and two for servo neg plus the feed. The middle is for positive (3 total - two for servo pos plus feed. Last the one on the left is for led signal - 2 plus a feed. The signal for the servos don’t need a bridge because they are controlled separately via mobiflight.Why? What this does is cut down on the number of wires needed for each instrument. So Instead of 6 wires (4 neg 2 positive 2 led signal) you only need 3 - one positive one negative and one signal led wires to each instrument. If you look this design is also on the large steam gauge as well that use the 28b stepper. In the case of using multiple steppers and leds (like the altimeter I see you’re working on) it can drastically cut down on the number of wires. Again these boxes are optional they do not have to be used at all. You may want to put your junctions elsewhere and that’s totally fine. Anyways I’m happy to send a picture of one fully wired when I get home from work if you want. Oh also btw I did get the models of the lcd. Thank you! You saved me some time hunting on grabcad. You can now delete those files thanks again!On Dec 29, 2022, at 12:14 PM, Trevor Olsen ***@***.***> wrote:﻿
I have a question @Kovinator - What do these do?

**Comment by Kovinator** [comment]
Haha apparently I can do math it’s not 6 it’s 8 cut down to 3. 😂 lol !On Dec 29, 2022, at 12:58 PM, Adam Pate ***@***.***> wrote:﻿They are optional “junction” boxes to combine common wires. You can take a pin header and solder them together to create a “bridge” that can then be hot glued in place inside each box. The one to the farthest right is common negative (5 wires total - 2 for led cathode and two for servo neg plus the feed. The middle is for positive (3 total - two for servo pos plus feed. Last the one on the left is for led signal - 2 plus a feed. The signal for the servos don’t need a bridge because they are controlled separately via mobiflight.Why? What this does is cut down on the number of wires needed for each instrument. So Instead of 6 wires (4 neg 2 positive 2 led signal) you only need 3 - one positive one negative and one signal led wires to each instrument. If you look this design is also on the large steam gauge as well that use the 28b stepper. In the case of using multiple steppers and leds (like the altimeter I see you’re working on) it can drastically cut down on the number of wires. Again these boxes are optional they do not have to be used at all. You may want to put your junctions elsewhere and that’s totally fine. Anyways I’m happy to send a picture of one fully wired when I get home from work if you want. Oh also btw I did get the models of the lcd. Thank you! You saved me some time hunting on grabcad. You can now delete those files thanks again!On Dec 29, 2022, at 12:14 PM, Trevor Olsen ***@***.***> wrote:﻿
I have a question @Kovinator - What do these do?

**Comment by CaptainBobSim** [comment]
Gotcha! That makes sense now.
I was initially worried that they'd interfere with the printability of the instruments but was just looking at things wrong :)
Clever thinking to have everything simplified at the instrument level - that'd allow much simpler wiring and potential maintenance improvements (be able to replace a servo easily)

Does it make sense to recess these like this to save material? Would this ruin things?
![image](https://user-images.githubusercontent.com/76851401/209992423-8ffb9cb8-a223-4dfd-a562-e3f7cc1d141c.png)

This is really awesome! I'd love to see it wired up.

**Comment by Kovinator** [comment]
Recessing is a great idea! I’ll have to get them updated! Thanks again !On Dec 29, 2022, at 1:15 PM, Trevor Olsen ***@***.***> wrote:﻿
Gotcha! That makes sense now.
I was initially worried that they'd interfere with the printability of the instruments but was just looking at things wrong :)
Clever thinking to have everything simplified at the instrument level - that'd allow much simpler wiring and potential maintenance improvements (be able to replace a servo easily)
Does it make sense to recess these like this to save material? Would this ruin things?

This is really awesome! I'd love to see it wired up.

**Comment by geologistluna** [comment]
Sorry to interrupt, but is this the backing of the small indicators?

**Comment by CaptainBobSim** [comment]
Yes

**Reply by geologistluna** [reply]
Is it located somewhere on github or did you remove it?

**Reply by Kovinator** [reply]
I'm looking into this. It does appear like it's missing which is most likely an error on my part. We went through a renaming scheme and it probably got overwritten or deleted. I will do another export from source.

**Reply by CaptainBobSim** [reply]
@Kovinator I consolidated the small instruments into two folders - EGT and Non EGT to save development effort (it's tricky fixing something 4 times)

**Reply by Kovinator** [reply]
Got the servo motor plate exported and submitted a pull request. should be all good now. Thanks to geologistluna for pointing it out!
